features-support/step_definitions/features.js

```diff
@@ -85,13 +92,13 @@ module.exports = function () {
         return world.browser.findElement(By.id(repositoryCongtrolsId));
 
         // Get the repo controls element.
-      }, handleErr(callback))
```

Comment:
I haven't explicitly tested it, no. I guess I will need to do that. I have however looked into the source and it seems to return a promise with .then and .catch functions. And thank you for you comments, much appreciated.